src/main/scala/nl/knaw/dans/easy/umd/Command.scala

```diff
@@ -59,8 +59,14 @@ object Command {
     new AutoDetectReader(new FileInputStream(file))
   }
 
-  private def requireUTF8(file: File, reader: AutoDetectReader) = Try {
-    require(reader.getCharset.toString == "UTF-8", s"encoding of $file must be UTF-8 but is ${reader.getCharset}")
+  val acceptedCharsets = Set("UTF-8", "ISO-8859-1")
```

Comment:
`ISO-8859-1` is *not* `US-ASCII`. It makes sense to also accept `US-ASCII`, as it is a subset of `UTF-8`, but Latin-1 is not.